adapters/handlers/rest/configure_api.go

```diff
@@ -1524,12 +1525,14 @@ func initializeCasbin(authConfig config.APIKey) (*casbin.SyncedCachedEnforcer, e
 	if err != nil {
 		return nil, fmt.Errorf("failed to create enforcer: %w", err)
 	}
-	// TODO: make configurable
-	enforcer.EnableCache(false)
+	enforcer.EnableCache(true)
 
-	// TODO: make configurable
 	enforcer.SetAdapter(fileadapter.NewAdapter("./rbac/policy.csv"))
-	enforcer.AddNamedMatchingFunc("g", "KeyMatch2", casbinutil.KeyMatch2)
+	if err := enforcer.LoadPolicy(); err != nil {
+		return nil, err
+	}
+	enforcer.AddNamedMatchingFunc("g", "keyMatch5", casbinutil.KeyMatch5)
+	enforcer.AddNamedMatchingFunc("g", "regexMatch", casbinutil.RegexMatch)
```

Comment:
could you explain what these do? :)

Reply:
surething , planning to do in next PR https://casbin.org/docs/function/